recipe/meta.yaml

```
@@ -1,19 +1,21 @@
 # There is no gnu version of orjson on Windows
 # because the simdutf8 does not get properly linked
 {% set name = "orjson" %}
-{% set version = "3.8.8" %}
+{% set version = "3.9.10" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: c096d7a523bae6ffb9c4a228ba4691d66113f0f2231579dc945523fbef09c6da
+  sha256: 9ebbdbd6a046c304b1845e96fbcc5559cd296b4dfd3ad2509e33c4d9ce07d6a1
 
 build:
   number: 0
-  skip: true  # [py<37 or s390x]
+  skip: true  # [py<38]
+  missing_dso_whitelist:
+    - '$RPATH/ld64.so.1'
```

```suggestion
  missing_dso_whitelist:  # [s390x]
    - '$RPATH/ld64.so.1'  # [s390x]
```
I guess it's only for s390x?